sw/device/lib/pinmux.c

```diff
@@ -80,4 +80,14 @@ void pinmux_init(void) {
   reg_offset = kTopEarlgreyPinmuxMioOutIoc9 << 2;
   mask = PINMUX_MIO_OUTSEL_0_OUT_0_MASK;
   mmio_region_write32(reg32, reg_offset, reg_value & mask);
+
+  // Configure USBDEV SENSE outputs to be high-Z (IOR0, IOR1)
+  reg32 =
+      mmio_region_from_addr(PINMUX0_BASE_ADDR + PINMUX_MIO_OUTSEL_0_REG_OFFSET);
+  reg_value = 2;  // high-Z value
+  reg_offset = kTopEarlgreyPinmuxMioOutIor0 << 2;
```

Comment:
Can you copy the comment that is before all the other instances of `<< 2`? It's a bit repetitive but it at least gives some clue as to the reason for this calculation.

At some point it might be worth refactoring `pinmux_init` to be table driven like the version in `silicon_creator`. Again, no need to block this PR though.